gitbom/Cargo.toml

```diff
@@ -8,10 +8,10 @@ license = "Apache-2.0"
 name = "gitbom"
 readme = "../README.md"
 repository = "https://github.com/git-bom/gitbom-rs"
-version = "0.1.1"
+version = "0.1.3"
```

Comment:
Also, I uploaded gitbom 0.1.2 up to crates.io before fixing the API, so this is now 0.1.3.